test/suite/test_compact11.py

```diff
@@ -0,0 +1,197 @@
+        blocks = b.group(1)
+        self.prout(f"parse_blkmods for '{uri}': {blocks}")
+        return blocks
+
+    def test_compact11(self):
+        # Create N tables.
+        uris = []
+        files = []
+        bitmaps = []
+        for i in range(self.num_tables):
+            uri = self.uri_prefix + f'_{i}'
+            uris.append(uri)
+            files.append(f'file:test_compact11_{i}.wt')
+            self.session.create(uri, self.create_params)
+
+        # Populate the first half of each table.
+        for uri in uris:
+            self.populate(uri, 0, self.table_numkv // 2, self.value_size)
+
+        # Write to disk.
+        self.session.checkpoint()
+
+        os.mkdir(self.backup_incr)
+        self.initial_backup = True
+        self.take_full_backup(self.backup_incr)
+        self.initial_backup = False
+
+        # Insert the latter 50% in each table.
+        for uri in uris:
+            self.populate(uri, self.table_numkv // 2, self.table_numkv, self.value_size)
+
+        # Write to disk.
+        self.session.checkpoint()
+
+        # Delete 50% of each file.
+        for uri in uris:
+            self.delete_range(uri, 50 * self.table_numkv // 100)
+
+        # Write to disk.
+        self.session.checkpoint()
+        for uri in files:
+            bitmaps.append(self.parse_blkmods(uri))
+
+        # Enable background compaction.
+        self.turn_on_bg_compact('free_space_target=1MB')
+
+        # Every time compaction makes progress, perform an incremental backup.
+        bytes_recovered = 0
+        while self.get_files_compacted(uris) < self.num_tables:
+            new_bytes_recovered = self.get_bytes_recovered()
+            if new_bytes_recovered != bytes_recovered:
+                self.prout('Compaction has made progress... ')
+                # Update the incremental backup ID from the parent class.
+                bytes_recovered = new_bytes_recovered
+               
+        assert self.get_files_compacted(uris) == self.num_tables
+        self.turn_off_bg_compact()
+        self.prout(f'Compaction has compacted {self.num_tables} tables.')
+
+        self.bkup_id += 1
+        self.take_incr_backup(self.backup_incr)
+        
+        self.prout('Comparing the original bitmaps with the new ones...')
+        for index, uri in enumerate(files):
+            new_bitmap = self.parse_blkmods(uri)
+            self.compare_bitmap(bitmaps[index], new_bitmap)
+
+        # Take a second full backup.
+        os.mkdir(self.backup_full)
+        self.take_full_backup(self.backup_full)
+
+        # Compare the backups.
+        for uri in uris:
+            self.compare_backups(uri, self.backup_incr, self.backup_full)
+
+        # Background compaction may have been inspecting a table when disabled, which is considered
+        # as an interruption, ignore that message.
+        self.ignoreStdoutPatternIfExists('background compact interrupted by application')
+
+if __name__ == '__main__':
+    wttest.run()
```

Comment:
```suggestion
    wttest.run()

```

Reply:
Update the incremental backup ID from the parent class.